lectures/markov_chains.md

```diff
@@ -998,43 +1022,67 @@ for i in range(n_state):
 plt.show()
 ```
 
-In fact, it converges faster given it is a more "predictable" dynamic
-
-We will come back to this very soon.
-
++++ {"user_expressions": []}
 
 ### Asymptotic Stationarity
 
-Sometimes the distribution $\psi_t = \psi_0 P^t$ of $X_t$ converges to the
-stationary distribution regardless of where we begin.
+Consider an irreducible stochastic matrix with unique stationary distribution $\psi^*$.
+
+Sometimes the distribution $\psi_t = \psi_0 P^t$ of $X_t$ converges to $\psi^*$ regardless of $\psi_0$.
 
 For example, we have the following result
 
 ```{prf:theorem}
 :label: strict_stationary
 
 Theorem: If there exists an integer $m$ such that all entries of $P^m$ are
-strictly positive, then $P$ has only one stationary distribution $\psi^*$ and
+strictly positive, then $P$ is irreducible, with unique stationary distribution $\psi^*$, and
 
 $$
     \psi_0 P^t \to \psi
     \quad \text{as } t \to \infty
 $$
 
 
-(See, for example, {cite}`haggstrom2002finite`. Our assumptions imply that $P$
-is irreducible and [aperiodic](https://en.wikipedia.org/wiki/Aperiodic_graph).)
 ```
 
-The convergence in the theorem is illustrated in the next figure
+
+See, for example, {cite}`sargent2023economic` Chapter 4.
+
++++ {"user_expressions": []}
+
+#### Example: Hamilton's Chain
+
+Hamilton's chain satisfies the conditions of the theorem because $P^2$ is everywhere positive:
 
 ```{code-cell} ipython3
 P = np.array([[0.971, 0.029, 0.000],
               [0.145, 0.778, 0.077],
               [0.000, 0.508, 0.492]])
 P = np.array(P)
+P @ P
+```
+
++++ {"user_expressions": []}
+
+Let's pick an initial distribution $\psi$ and trace out the sequence of distributions $\psi P^t$.
+
+First, we write a function to simulate the sequence of distributions for `n` period
+
```

Comment:
@HumphreyYang I suggest we avoid the use of the word "simulate" here and in the name of the function, since there is no randomness involved.  (We are just iterating on the deterministic vector difference equation $\psi_{t+1} = \psi_t P$.)

Reply:
Many thanks @jstac for picking this up.  I have changed the function name to `iterate_ψ`.